File: javascript/src/models/airwallexConfig.ts
// this file is @generated

export interface AirwallexConfig {
  secret: string;
}

export const AirwallexConfigSerializer = {
  _fromJsonObject(object: any): AirwallexConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: AirwallexConfig): any {
    return {
      secret: self.secret,
    };
  },
};

File: rust/src/models/airwallex_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct AirwallexConfig {
    pub secret: String,
}

impl AirwallexConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}
